CONTRIBUTING.md

```diff
@@ -326,10 +326,13 @@ Here's an example:
 
 ```python
 @check_figures_equal()
-def test_my_plotting_case(fig_ref, fig_test):
+def test_my_plotting_case():
   "Test that my plotting function works"
+  fig_ref = Figure()
   fig_ref.grdimage("@earth_relief_01d_g", projection="W120/15c", cmap="geo")
+  fig_test = Figure()
   fig_test.grdimage(grid, projection="W120/15c", cmap="geo")
+  return fig_ref, fig_test
```

Comment:
I like the new design. It's more readable. 

Just one question: Is it allowed to use other names for the figures, like `fig1` and `fig2`?

Reply:
> Just one question: Is it allowed to use other names for the figures, like `fig1` and `fig2`?

Yes, the name doesn't matter, as long as they're `pygmt.Figure()` instances, or anything with a `savefig` method actually (this refactor works with matplotlib too).

Reply:
That's great.